OWNERS

```diff
@@ -1,18 +1,14 @@
 approvers:
-  - abhi-g
   - jlewi
-  - richardsliu
   - joeliedtke
   - animeshsingh
   - rmgogogo
-  - janeman98
 reviewers:
   - aronchick
   - carmine
```

Comment:
We can remove @carmine I think; I don't think he's involved in Kubeflow anymore.